frontend/src/locales/en/common.json

```diff
@@ -0,0 +1,3 @@
+{
+    "loading_message": "Loading content"
```

Comment:
What is this for? Just curious :)

Reply:
It's not used anywhere so I'll remove it.